scripts/publish-rc-chart.sh

```diff
@@ -13,12 +13,10 @@ cp -r -L chart/osiris /tmp/osiris-edge
 cd /tmp/osiris-edge
 
 sed -i "s/^name:.*/name: osiris-edge/g" Chart.yaml
-sed -i "s/^version:.*/version: 0.0.1-$DATE-$GIT_VERSION/g" Chart.yaml
-sed -i "s/^appVersion:.*/appVersion: $GIT_VERSION/g" Chart.yaml
 sed -i "s/^  tag:.*/  tag: $GIT_VERSION/g" values.yaml
 sed -i "s/^  pullPolicy:.*/  pullPolicy: IfNotPresent/g" values.yaml
 
 helm dep build .
-helm package .
+helm package --version 0.0.1-$DATE-$GIT_VERSION --app-version $GIT_VERSION .
```

Comment:
In newer versions of Helm, this will wrap the values of the `version` and/or `appVersion` fields in quotes if they are values that a YAML parser might later mistake for a number. This is _also_ a more elegant alternative to using `sed` to insert version information into `Chart.yaml`.